### ISSUE TITLE
Where are the container images coming from / how to publish images?

### ISSUE THREAD

**Issue by theogravity** [codius/manifest#13]
It's not clear from the spec as to where the images come from.

`"image": "hello-world@sha256:f5233545e43561214ca4891fd1157e1c3c563316ed8e237750d59bde73361e77"`

Where is the `hello-world` image stored (eg docker hub)? 

Is there an example repository for building images?

**Comment by theogravity** [comment]
Ah, it does look like docker hub is the default.

Searching for the hello-world image hash yields a lot of results on google.

**Comment by sharafian** [comment]
Yeah, it uses the same format as docker to pull images. So `hello-world@sha256:...` pulls from docker hub but `docker.example.com/hello-world@sha256:...` pulls from a docker registry server at `docker.example.com`.
